recipe/meta.yaml

```diff
@@ -90,24 +87,15 @@ outputs:
       files:
         - tests
       commands:
-        # Verify b2 and bjam executables work
```

Comment:
can you add this comment in the recipe itself?
Had to remove some old-style tests because of this build issue introduced with 1.74.0 (but fixed in 1.76.0):
https://github.com/boostorg/build/issues/677